{
  "comments": [
    {
      "key": {
        "uuid": "d5be4ea4_c90c0071",
        "filename": "test/sad_test.cc",
        "patchSetId": 3
      },
      "lineNbr": 112,
      "author": {
        "id": 1002219
      },
      "writtenOn": "2018-03-28T23:15:26Z",
      "side": 1,
      "message": "is this and the stride change needed now or was it just masking the sse2 failure in this test?",
      "revId": "39de45d3cc178012cd5e8e17a2f9bcae2f858299",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    }
  ]
}